gtdynamics/universal_robot/sdf.cpp

```diff
@@ -75,19 +75,18 @@ JointParams ParametersFromSdfJoint(const sdf::Joint &sdf_joint) {
 Pose3 GetJointFrame(const sdf::Joint &sdf_joint,
                     const LinkSharedPtr &parent_link,
                     const LinkSharedPtr &child_link) {
-  auto frame = sdf_joint.PoseFrame();
-  if (frame == "" || frame == child_link->name()) {
-    if (sdf_joint.Pose() == ignition::math::Pose3d())
+  auto frame = sdf_joint.PoseRelativeTo();
+  Pose3 sdf_joint_pose = Pose3FromIgnition(sdf_joint.RawPose());
+  if (frame.empty() || frame == child_link->name()) {
+    if (sdf_joint.RawPose() == ignition::math::Pose3d()) {
```

Comment:
Why even have this case? If it's identity we multiply with identity. So kill.

Reply:
Yeah I found this strange too when I looked at this (it was pre-existing).